CASC/PortfolioMode.cpp

```diff
@@ -632,12 +632,7 @@ void PortfolioMode::runSlice(Options& strategyOpt)
 {
   System::registerForSIGHUPOnParentDeath();
   UIHelper::portfolioParent=false;
-
-  env.timer->reset();
-  env.timer->start();
-
-  Timer::resetInstructionMeasuring();
-  Timer::setLimitEnforcement(true);
+  Timer::reinitialise();
```

Comment:
This must be called here as we are in a new process. The parent process' timer thread will keep ticking, but it won't kill us or update our instruction counts.